Tribler/Core/Modules/MetadataStore/OrmBindings/channel_metadata.py

```diff
@@ -450,7 +452,7 @@ def to_simple_dict(self):
                 "subscribed": self.subscribed,
                 "votes": self.votes,
                 "status": self.status,
-                "updated": self.timestamp,
+                "updated": int((self.torrent_date - epoch).total_seconds() * 1000.0),
```

Comment:
You don't even need to `*1000.0`. `torrent_date` is seconds-granularity (as in .torrent files).